packages/glimmer-runtime/tests/updating-test.ts

```diff
@@ -1371,8 +1371,8 @@ test("<option selected> is normalized and updated correctly", function() {
   let expectedInitialTokens = `
     <select multiple="">
       <option>0</option>
-      <option selected="">1</option>
-      <option selected="">2</option>
+      <option>1</option>
```

Comment:
Changing these seem wrong, it implies that the options are not selected when they should be (and this was the point of the test originally).